api/AudioBuffer.json

```diff
@@ -0,0 +1,659 @@
+{
+  "api": {
+    "AudioBuffer": {
+      "AudioBuffer": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioBuffer/AudioBuffer",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "58"
+              },
+              {
+                "version_added": "55",
+                "version_removed": "57",
+                "notes": "Supported the context parameter, which has now been removed from the spec."
+              }
+            ],
+            "edge": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "53"
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "42",
+                "version_removed": "44",
+                "notes": "Supported the context parameter, which has now been removed from the spec."
+              }
+            ],
+            "safari": {
+              "version_added": null
+            },
+            "webview_android": [
+              {
+                "version_added": "58"
+              },
+              {
+                "version_added": "55",
+                "version_removed": "57",
+                "notes": "Supported the context parameter, which has now been removed from the spec."
+              }
+            ],
+            "firefox_android": {
+              "version_added": "53"
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "opera_android": [
+              {
+                "version_added": "45"
+              },
+              {
+                "version_added": "42",
+                "version_removed": "44",
+                "notes": "Supported the context parameter, which has now been removed from the spec."
+              }
+            ],
+            "safari_ios": {
+              "version_added": null
+            },
+            "chrome_android": [
+              {
+                "version_added": "58"
+              },
+              {
+                "version_added": "55",
+                "version_removed": "57",
+                "notes": "Supported the context parameter, which has now been removed from the spec."
+              }
+            ]
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "duration": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioBuffer/duration",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "35"
+              },
+              {
```

Comment:
I don't think this property has been prefixed (only the interface). That is WebkitAudoBuffer but not AudioBuffer.webkitduration (or .-webkit-duration) I think you should merge prefix/unprefix both entries

Same thing everywhere (I don't write a comment for each of them).